{
	"schema_version": "1.6.0",
	"id": "RHSA-2022:6155",
	"modified": "2025-01-20T10:04:02Z",
	"published": "2024-09-30T14:23:20Z",
	"related": [
		"CVE-2022-23772",
		"CVE-2022-24675",
		"CVE-2022-28327"
	],
	"summary": "Red Hat Security Advisory: Red Hat OpenShift Data Foundation 4.11.0 security, enhancement & bugfix update",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Red Hat:openshift_data_foundation:4.11::el8",
				"name": "mcg",
				"purl": "pkg:rpm/redhat/mcg"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:5.11.0-22.el8"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2022:6155.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:openshift_data_foundation:4.11::el8",
				"name": "mcg-redistributable",
				"purl": "pkg:rpm/redhat/mcg-redistributable"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:5.11.0-22.el8"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2022:6155.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2022:6155"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com//documentation/en-us/red_hat_openshift_data_foundation/4.11/html/4.11_release_notes/index"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/security/updates/classification/#moderate"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2027849"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2053532"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2077688"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2077689"
		},
		{
			"type": "ADVISORY",
			"url": "https://security.access.redhat.com/data/csaf/v2/advisories/2022/rhsa-2022_6155.json"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2022-23772"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2022-23772"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2022-23772"
		},
		{
			"type": "ARTICLE",
			"url": "https://groups.google.com/g/golang-announce/c/SUsQn0aSgPQ"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2022-24675"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2022-24675"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2022-24675"
		},
		{
			"type": "ARTICLE",
			"url": "https://groups.google.com/g/golang-announce/c/oecdBNLOml8"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2022-28327"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2022-28327"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2022-28327"
		}
	]
}
